.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,6 +20,10 @@ For new features there should be at least one test showing the expected behavior
 
 Are there any side effects that we should be aware of?
 
+## Release notes
+
+Replace this text with a short note on what will change if this pull request is merged in which case this will be included in the release notes.
+
```

Comment:
Since we are using the release_notes.md should we just have a checkmark like

- [ ] Added release note to release_notes.md (optional)

Reply:
The final agreement was to go with a section in the PR template not the separate md file. The main reason being git conflicts

Reply:
Ah ic yeah sure